runtime/development/src/lib.rs

```diff
@@ -936,6 +936,56 @@ impl PoolUpdateGuard for UpdateGuard {
 	}
 }
 
+pub struct CurrencyPriceSource;
+impl CurrencyPrice<CurrencyId> for CurrencyPriceSource {
+	type Rate = Rate;
+	type Moment = Moment;
+
+	fn get_latest(
```

Comment:
This is complex enough to warrant a test, either here or as broken up into an accessor on the pallet

Reply:
Yeah the rewriting to an accessor on the pallet already happened, test is still in progress :)